cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -207,16 +204,50 @@ func RemoveContents(dir string) error {
 	return nil
 }
 
+type isRetriable func(error) bool
+type function func() error
+
+func retryOnError(shouldRetry isRetriable, f function) error {
```

Comment:
Do we need this function only here or could be reused in other parts of the code?

Reply:
I did not find a place where we need it. Do you suggest otherwise? Note: this is specific optimization. It is perfectly ok to use the former function in other components.